mrp_bom_line_net_qty/models/mrp_bom_line.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2023 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class MrpBomLine(models.Model):
+    _inherit = "mrp.bom.line"
+
+    product_qty_net = fields.Float(
+        string="Net quantity",
+        help="Weight after any preparation",
```

Comment:
i've been a bit quick on the review.

"Weight" and "preparation" are really linked in your domain

Reply:
do you mean that i should have a more generic help message ?
do you have a proposal ? :)

Reply:
yes like this. 
you may have liters or unit as input components instead of "weight"

Reply:
I chose "Quantity after process" for the help message